dell/inspiron/7405/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, ... }:
+
+with lib;
+
+{
+  imports = [
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  boot = {
+    initrd.availableKernelModules = [ "nvme" "xhci_pci" "ahci" "usb_storage" "usbhid" "sd_mod" ];
```

Comment:
Usually we don't have kernel modules that are also detected by `nixos-generate-config` since names might change over time and people might swap different storage devices in their laptops.